scripts/Dockerfile.in

```diff
@@ -63,7 +63,7 @@ RUN /alpine-extractor.sh musl openssl-dev openssl-libs-static linux-headers
 
 ## The bash container fetches bash-static (to be used as bash) from Debian repo.
 
-FROM debian:11.4 AS bash
+FROM debian:bookworm-slim AS bash
```

Comment:
we had some hiccups with this change, have you verified this is ok?

Reply:
Tested in a cluster and confirmed it working.